packages/docs-theme/src/components/documentation.tsx

```diff
@@ -156,8 +168,13 @@ export class Documentation extends React.PureComponent<DocumentationProps, Docum
         // build up static map of all references to their page, for navigation / routing
         this.routeToPage = {};
         eachLayoutNode(this.props.docs.nav, (node, parents) => {
-            const { reference } = isPageNode(node) ? node : parents[0]!;
-            this.routeToPage[node.route] = reference;
+            if (isNavSection(node)) {
+                return;
+            } else if (isPageNode(node)) {
```

Comment:
Is the _"components"_ navigation section passing the `isPageNode` check? Could that be a bug in Documentalist or are we setting up the _components_ section in a weird/unexpected way?

Reply:
going on intuition haven't dug into the code much here but I believe "components" is a page node because it has sub-pages - `/docs/#core/components/breadcrumbs`